ui/src/app/pages/homepage/data-access-requirements.tsx

```diff
@@ -36,6 +39,7 @@ import {
   bypassAll,
   getAccessModuleConfig,
   getAccessModuleStatusByName,
```

Comment:
these 2 are the same function (one calls the other) but they have different signatures